pep/app/services/lang.ts

```diff
@@ -14,21 +16,36 @@ export default class LangService extends Service {
 
     @tracked currentLanguage: LanguageCode = LanguageCode.enUS;
 
+    get availableLanguages() {
+        return Languages.map((lang) => ({
+            ...lang,
+            label: this.intl.t(lang.label)
+        }));
+    }
+
     /**
      * Initialize the app with the user's current language
      */
-    setup() {
+    async setup() {
         // TODO eventually possibly allow setting via ?lang={code} app-level query param
         this.currentLanguage = this.currentUser.preferences?.lang ?? LANG_EN_US.code;
+        await this.loadLanguage(this.currentLanguage);
         this.intl.setLocale(this.currentLanguage);
     }
 
+    async loadLanguage(lang: LanguageCode) {
+        const translations = await fetch(`/translations/${lang}.json`);
```

Comment:
we should do this, to allow for optimal browser caching/cache-busting of the translation files:

https://ember-intl.github.io/ember-intl/versions/master/docs/guide/asynchronously-loading-translations#fingerprinting

should be pretty straightforward, I'd probably just store the full uninterpolated paths in the lang object constants themselves, so that the fingerprinting replacement happens automatically e.g. 

```ts
export const LANG_FR_FR: Language = {
    code: LanguageCode.frFR,
    label: 'language.codes.frFR',
   path: 'assets/translations/fr-fr.json'
};
```
I think we need the fingerprinting rewriting of the url paths to happen in any case, because we'll want the translations files served from the S3 CDN, not the local fastboot server. The fingerprinting process prepends that assets base url for you as well.